### ISSUE TITLE
SwiftUI Image focus on faces

### ISSUE THREAD

**Issue by otymartin** [BeauNouvelle/FaceAware#37]
@BeauNouvelle How can we use this magic in SwiftUI's `Image` view?

**Comment by BeauNouvelle** [comment]
Since this is just an extension, you should be able to wrap a regular UIImageView in UIViewRepresentable and call the appropriate methods in there.

I haven't tested it though.

I'm looking at writing a replacement for this library that works with SwiftUI!
When time permits!

**Comment by BeauNouvelle** [comment]
If the above suggestion doesn't work, please let me know and I'll look into improving it.

**Comment by otymartin** [comment]
Thanks, @otymartin Will let you know once I implement it - generally, I'm just eliminating UIKit in favour of native SwiftUI so I wanted to avoid it.

**Comment by BeauNouvelle** [comment]
Yeah, I would love to redo all this. I'm no longer building new projects using UIKit.

**Comment by otymartin** [comment]
@BeauNouvelle Good to hear - looking forward to whatever magic you produce :)
